bin/setup

```diff
@@ -22,7 +22,7 @@ FileUtils.chdir APP_ROOT do
   puts '== Installing dependencies =='
   system! 'gem install bundler --conservative'
 
-  puts "\n== Running submodule update ==ß"
+  puts "\n== Running submodule update =="
```

Comment:
IDK where this came from, since we don't use this letter in English. Também não usamos essa letra em português. 😆

cc @gustavohc07

Reply:
Ha!! Thank you for that! That is the result of using a non-mac keyboard and trying to save the open file... Sometimes I hit `alt+s` instead of `cmd+s` to save a file and `ß` appears.
Por um momento eu pensei que você fosse brasileiro ou português! 😆